Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Build Stage
```

Comment:
Is it possible to separate this into a multi-stage build?

Reply:
Just did! I initially had problems with GLIBC when I tried, but switching to Ubuntu 22.04 resolved it.